exercises/practice/ocr-numbers/.docs/instructions.md

```diff
@@ -62,60 +62,18 @@ Is converted to "1234567890"
 Update your program to handle multiple numbers, one per line. When converting several lines, join the lines with commas.
 
 ```text
-    _  _
+    _  _ 
   | _| _|
   ||_  _|
-
-    _  _
-|_||_ |_
+         
+    _  _ 
+|_||_ |_ 
   | _||_|
-
- _  _  _
+         
+ _  _  _ 
   ||_||_|
   ||_| _|
-
+         
```

Comment:
The PR check fails due to all those extra trailing spaces.

Reply:
Should be fixed now.